use-cases/using-SPDK-vhostuser-and-kata.md

```diff
@@ -176,26 +176,44 @@ $ sudo scripts/rpc.py vhost_create_blk_controller vhostblk0 Malloc0
 Considering the OCI specification and characteristics of vhost-user device,
 Kata has chosen to use Linux reserved the block major range `240-254`
 to map each vhost-user block type to a major. Also a specific directory is
-used for vhost-user devices. The base directory is a configurable value,
+used for vhost-user devices.
+
+The base directory for vhost-user device is a configurable value,
 with the default being `/var/run/kata-containers/vhost-user`. It can be
 configured by parameter `vhost_user_store_path` in [Kata TOML configuration file](https://github.com/kata-containers/runtime/blob/master/README.md#configuration).
 
-The reset of the path `block` is used for block device; `block/sockets` is where
-we expect vhost-user sockets to live; `block/sockets` is where simulated block
-device node for vhost-user devices to live.
+Currently, storage device in vhost-user type is not enabled by default, so
+user should enable it explicitly inside Kata TOML configuration file by
+parameter `enable_vhost_user_store = true`. Since SPDK vhost-user target
```

Comment:
This option is only available for QEMU. Since the whole document only refers to QEMU, I do think it would be worth adding a note at the top of the document stating that this guide only applies to QEMU.